src/MonoTorrent/MonoTorrent.Client/Settings/TorrentSettingsBuilder.cs

```diff
@@ -99,6 +99,11 @@ public class TorrentSettingsBuilder
         /// </summary>
         public int WebSeedSpeedTrigger { get; set; }
 
+        /// <summary>
+        /// If set to false then root folder will be not created for multi-files torrents
```

Comment:
```suggestion
        /// If set to false then root folder will be not created for multi-files torrents, otherwise all files will be placed inside a root folder named <see cref="TorrentManager.Name"/>. Defaults to true.
```